cleaners/winrar.xml

```diff
@@ -47,8 +49,8 @@
     <action command="winreg" path="HKCU\Software\WinRAR SFX"/>
   </option>
   <option id="temp">
-    <label>Temp Files</label>
-    <description>Deletes the Temp Files</description>
+    <label>Temporary files</label>
+    <description>Delete the temporary files</description>
```

Comment:
These two are translated. This is good. Tanks

Reply:
No prob! I get better... ;-)